kodilla-rps/src/main/java/com/kodilla/rps/RpsRunner.java

```diff
@@ -9,72 +8,34 @@ public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
```

Comment:
dobrze by było tak to zamodelować, żeby w `main` było tylko:
```
User user = new User(..);
Game game = new Game(user);
```
+ jakiś obiekt opakowujący scanner, który zaczyta potrzebne dane do rozpoczęcia gry